.github/workflows/dis-syncroot-admin-release.yml

```
@@ -0,0 +1,72 @@
+  publish-admin-syncroot:
+    name: Publish admin syncroot artifacts
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
+
+      - name: Validate admin syncroot structure
+        # Local path (not the pinned published ref): this PR adds the `environments`
+        # input to verify-syncroot, so the workflow must run the in-tree version.
+        uses: ./actions/flux/verify-syncroot
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/actions/flux/verify-syncroot": line 5: unexpected key "type" for definition of input "environments"

(action)